exampleCourse/courseInstances/SectionA/assessments/groupWork/template/infoAssessment.json

```diff
@@ -40,17 +40,17 @@
             "credit": 100
         }
     ],
+    "canView": ["Manager", "Reflector", "Recorder", "Contributor"],
     "zones": [
         {
             "title": "Fundamental questions",
             "bestQuestions": 3,
+            "canSubmit": ["Recorder"],
             "questions": [
                 {
                     "id": "demo/demoNewton-page1",
                     "points": 1, 
-                    "maxPoints": 5, 
-                    "canView": ["Manager", "Reflector", "Recorder", "Contributor"],
-                    "canSubmit": ["Recorder"]
+                    "maxPoints": 5
```

Comment:
Need to change this back to the original layout where all permissions are specific to each question, for the sake of clarity. Zone & assessment level syncing can be shown in other assessments and in documentation